apps/ringtones/js/share.js

```diff
@@ -228,13 +227,16 @@ function handleShare(data, callback) {
         // Show a notification indicating success, and then close it immediately
         // so it doesn't stink up the notifications tray! XXX: This UX isn't
         // great; we should turn this into a transient notification when we can.
-        new Notification(
+        NotificationHelper.send(
           songtitle,
-          {body: navigator.mozL10n.get(
-            details.setAsDefault ? 'set-default-tone' : 'created-tone'
-          )}
-        ).close();
-        callback('save', details);
+          { bodyL10n:
+            details.setAsDefault ? 'set-default-tone' : 'created-tone',
```

Comment:
Nit: indent this by 2 more spaces, please. Or move some of it onto the previous line, e.g.:

```
          { bodyL10n: details.setAsDefault ? 'set-default-tone' :
                      'created-tone',
```